examples/summarization/bart/evaluate_cnn.py

```diff
@@ -51,15 +53,18 @@ def _run_generate():
     parser.add_argument(
         "output_path", type=str, help="where to save summaries",
     )
+    parser.add_argument(
+        "model_name", type=str, default="bart-large-cnn", help="like bart-large-cnn",
+    )
     parser.add_argument(
         "--device", type=str, required=False, default=DEFAULT_DEVICE, help="cuda, cuda:1, cpu etc.",
     )
     parser.add_argument(
         "--bs", type=int, default=8, required=False, help="batch size: how many to summarize at a time",
     )
     args = parser.parse_args()
-    lns = [" " + x.rstrip() for x in open(args.source_path).readlines()]
-    generate_summaries(lns, args.output_path, batch_size=args.bs, device=args.device)
+    examples = [" " + x.rstrip() for x in open(args.source_path).readlines()]
```

Comment:
Should we maybe add " " as a prefix parameter to the `task_specific_params` dictionary in `bart-large-cnn` and `bart-tiny-random` config as "summarize :" is added for `t5-base` (https://s3.amazonaws.com/models.huggingface.co/bert/t5-base-config.json). This change and by using AutoModelWithLMHead and AutoTokenizer the t5 and bart summarization example scripts would actually be exactly the same and could be merged

Reply:
yeah!(Future PR)

Reply:
Can you add so that I don't screw it up?
Affected Files:
```
s3://models.huggingface.co/bert/sshleifer/bart-tiny-random/config.json
s3://models.huggingface.co/bert/facebook/bart-large-cnn/config.json
s3://models.huggingface.co/bert/facebook/bart-large-xsum/config.json
s3://models.huggingface.co/bert/facebook/bart-large/config.json
```

Reply:
Yes I can do it! Can all these 4 models be used for summarization, then I'd add `prefix=" "` under `task_specific_params: { summarization: { prefix=" ", } }` ? Or should we add `prefix=" "` in the default configs, so that it's `prefix = " "` regardless of the specific task for Bart?

Reply:
They can all be used.
I like "prefix = " " regardless of the specific task for Bart". 
thx!

Reply:
done